src/Evolution/Systems/GrMhd/ValenciaDivClean/PrimitiveFromConservative.cpp

```diff
@@ -102,26 +105,34 @@ void PrimitiveFromConservative<OrderedListOfPrimitiveRecoverySchemes,
   rest_mass_density_times_lorentz_factor =
       get(tilde_d) / get(sqrt_det_spatial_metric);
 
+  disable_floating_point_exceptions();
```

Comment:
I'm not sure I'm convinced about providing provision in these functions for using nan as a flag to indicate that it needs to be initialized to an arbitrary guess. It seems unsafe, because many functions we have initialize to nan only in debug and just permit malloc garbage to populate data structures on release compiles, so this function would 'fix' a forgotten initialization of pressure only in debug cases when the first entry is uninitialized (but not if, say, only the first entry was initialized).

Reply:
I am puzzled by why this is needed; the pressure should be initialized before this is called

Reply:
Yes, I think this is a test issue (I realized this yesterday after thinking about Jordan's comment). So I think the thing to do is fix the test.